safe_transaction_service/history/tests/test_views.py

```diff
@@ -69,6 +69,15 @@ def test_about_view(self):
         response = self.client.get(url, format="json")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_swagger(self):
+        url = reverse("schema-json", args=(".json",))
+        response = self.client.get(url, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        url = reverse("schema-swagger-ui")
+        response = self.client.get(url, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Out of curiosity: was the swagger interface [here](https://github.com/safe-global/safe-transaction-service/pull/1526) returning an error code or was it returning `200` while rendering an error message? 🤔

Reply:
It was returning `200`, indeed. Error was on client side when requesting the `json` schema